tests/unit/expected_config/config.conf

```diff
@@ -64,6 +64,9 @@ configuration:
     enable: true
     expireTime: 6s
     maxRetryTimes: 4
+logger:
+  AMF:
+    debugLevel: info
```

Comment:
In the JIRA ticket I also proposed setting the `ReportCaller` to true when we go to `debug` mode. It made my life much easier when I was debugging the upstream.
Should we include it this feature?

Reply:
Are you sure reportCaller is still a thing? I can see no reference to it in the upstream logger code:

Reference: https://github.com/omec-project/util/blob/main/logger/logger_config.go

Reply:
Indeed, the report caller config option got removed part of the move from logrus to zap. For reference here's the PR upstream: https://github.com/omec-project/util/pull/45

Reply:
Yes,  
https://github.com/omec-project/util/blob/main/logger/logger.go#L47 adds the caller function and line to the log output. So, this is already available.